views/personal/maritalStatus-edit.pug

```diff
@@ -10,27 +10,18 @@ block content
   div
     p Please choose the correct status from the options listed below.
 
+  //- this is messy (data && data.personal && data.personal.maritalStatus) but pug will error if a property doesn't exist. We could handle this on the controller level, and pass this to the route? Will help keep it a bit cleaner here
```

Comment:
Yeah, I hate that

Reply:
>Will help keep it a bit cleaner here

Trying to keep the view as clean as possible seems reasonable.